aiven/resource_kafka_acl_test.go

```diff
@@ -44,7 +44,12 @@ func TestAccAivenKafkaACL_basic(t *testing.T) {
 				ExpectError: regexp.MustCompile("invalid value for username"),
 			},
 			{
-				Config:             testAccKafkaACLWildcardResource(rName),
```

Comment:
Can we keep `testAccKafkaACLWildcardResource` test as well?

Reply:
It is still in there, line 52, just the name was modified